src/Elmish.WPF/ViewModel.fs

```diff
@@ -7,6 +7,7 @@ open System.Collections.ObjectModel
 open System.ComponentModel
 open System.Windows
 open Elmish.WPF
+open System.Windows.Threading
```

Comment:
Could you move this up below `System.Windows`?

Reply:
OK.

Reply:
No need for this one anymore if we don't use `Dispatcher` in the module.